platform/ios/app/MBXState.m

```diff
@@ -0,0 +1,74 @@
+#import "MBXState.h"
+
+NSString *const MBXCamera = @"MBXCamera";
+NSString *const MBXUserTrackingMode = @"MBXUserTrackingMode";
+NSString *const MBXShowsUserLocation = @"MBXShowsUserLocation";
+NSString *const MBXDebugMaskValue = @"MBXDebugMaskValue";
+NSString *const MBXShowsZoomLevelOrnament =  @"MBXShowsZoomLevelOrnament";
+NSString *const MBXShowsTimeFrameGraph = @"MBXShowsFrameTimeGraph";
+NSString *const MBXDebugLoggingEnabled = @"MGLMapboxMetricsDebugLoggingEnabled";
+NSString *const MBXShowsMapScale = @"MBXMapShowsScale";
+NSString *const MBXMapShowsHeadingIndicator = @"MBXMapShowsHeadingIndicator";
+NSString *const MBXMapFramerateMeasurementEnabled = @"MBXMapFramerateMeasurementEnabled";
+NSString *const MBXReuseQueueStatsEnabled = @"MBXReuseQueueStatsEnabled";
+
+@interface MBXState()
+
+@end
+
+@implementation MBXState
+
+- (void)encodeWithCoder:(NSCoder *)coder
+{
+    [coder encodeObject:_camera forKey:MBXCamera];
+    [coder encodeObject:[NSNumber numberWithInt:_userTrackingMode] forKey:MBXUserTrackingMode];
```

Comment:
Xcode 11b1 warns about this implicit conversion from `NSUInteger` to `int`. [`MGLUserTrackingMode`](https://github.com/mapbox/mapbox-gl-native/blob/cf153c34ee8d77bbb0f53936eec4f58f26a0b761/platform/ios/src/MGLMapView.h#L78) is defined as an `enum` of `NSUInteger`, so this conversion should use `+[NSNumber numberWithUnsignedInteger:]`.